# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/.github/workflows/publish.yml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/.github/workflows/publish.yml b/gfx/wgpu/.github/workflows/publish.yml
deleted file mode 100644
--- a/gfx/wgpu/.github/workflows/publish.yml
+++ /dev/null
@@ -1,47 +0,0 @@
-name: Publish
-
-on:
-  push:
-    branches:
-      - gecko
-
-env:
-  RUSTFLAGS: --cfg=web_sys_unstable_apis
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout the code
-        uses: actions/checkout@v2
-        with:
-          persist-credentials: false
-
-      - name: Install Rust WASM toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          target: wasm32-unknown-unknown
-
-      - name: Build the examples
-        run: cargo build --release --target wasm32-unknown-unknown --examples
-
-      - name: Install wasm-bindgen-cli
-        run: cargo install wasm-bindgen-cli
-
-      - name: Generate JS bindings for the examples
-        run: |
-          for i in target/wasm32-unknown-unknown/release/examples/*.wasm;
-          do
-            wasm-bindgen --no-typescript --out-dir target/generated --web "$i";
-          done
-
-      - name: Deploy the examples
-        uses: JamesIves/github-pages-deploy-action@releases/v3
-        with:
-          ACCESS_TOKEN: ${{ secrets.WEB_DEPLOY }}
-          FOLDER: target/generated
-          REPOSITORY_NAME: gfx-rs/wgpu-rs.github.io
-          BRANCH: master
-          TARGET_FOLDER: examples/wasm